code/components/jomjol_wlan/read_wlanini.cpp

```diff
@@ -119,6 +120,19 @@ void LoadWlanFromFile(std::string fn, char *&_ssid, char *&_password, char *&_ho
             }
         }
 
+        if ((zerlegt.size() > 1) && (toUpper(zerlegt[0]) == "HTTP_USERNAME")){
+            http_username = zerlegt[1];
+            if ((http_username[0] == '"') && (http_username[http_username.length()-1] == '"')){
+                http_username = http_username.substr(1, http_username.length()-2);
+            }
+        }
+
+        if ((zerlegt.size() > 1) && (toUpper(zerlegt[0]) == "HTTP_PASSWORD")){
```

Comment:
The formatting in this block is also a bit off. Looks like opening curly braces are supposed to go on newlines around here.